trlx/trainer/accelerate_base_trainer.py

```diff
@@ -78,19 +81,39 @@ def __init__(self, config, **kwargs):
             dist_config = get_distributed_config(self.accelerator)
             config_dict["distributed"] = dist_config
             init_trackers_kwargs = {}
-            if "wandb" in config.train.trackers:
+
+            if config.train.tracker not in ("wandb", "tensorboard"):
+                raise ValueError(
+                    f"Only supported trackers are wandb and tensorboard, got {config.train.tracker}"
+                )
+
+            if config.train.tracker == "wandb":
                 init_trackers_kwargs["wandb"] = {
                     "name": run_name,
                     "entity": self.config.train.entity_name,
                     "group": self.config.train.group_name,
                     "tags": ["/".join(get_git_tag())],
                     "mode": "disabled" if os.environ.get("debug", False) else "online",
                 }
-            self.accelerator.init_trackers(
-                project_name=self.config.train.project_name,
-                config=config_dict,
-                init_kwargs=init_trackers_kwargs,
-            )
+
+                self.accelerator.init_trackers(
+                    project_name=self.config.train.project_name,
+                    config=config_dict,
+                    init_kwargs=init_trackers_kwargs,
+                )
+            else:
```

Comment:
Thanks!
Could you add back a comment explaining what this branch is for and the flattening?

Reply:
Sure! 
- I had run into problems running trlx without wandb (I don't have an account as of now), and found an already opened issue precisely on this. The branch has minor modifications (which don't change the previous interface for wandb users) to allow tensorboard tracking
- The only tricky part is that wandb is pretty fancy and take nested dicts as logging params, this is not the case for tensorboard, hence the experiment config is fully flattened, and the only list is simply split apart (for the same reason)
Do let me know if anything is not clear or if I should add comments in the tensorboard specific logging

Reply:
Oh sorry, by branch I meant the `else` branch - i.e a short comment like
`else: # tracker == 'tensorboard' ` and `# flatten config for tensorboard, flatten lists in hparams into flattened config`